example/full/instance.properties

```diff
@@ -697,6 +697,12 @@ sleeper.default.parquet.dictionary.encoding.sortkey.fields=false
 # Whether dictionary encoding should be used for value columns in the Parquet files.
 sleeper.default.parquet.dictionary.encoding.value.fields=false
 
+# The length in bytes to truncate binary values if possible in a column index.
```

Comment:
Not sure that this needs to say "if possible". This comment could be extended to include the parquet property that this is used to set (`parquet.columnindex.truncate.length`?).